pkgs/applications/science/astronomy/phd2/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pkgconfig, cmake, gtk3, wrapGAppsHook,
+  wxGTK30-gtk3, curl, gettext, glib, indilib, libnova }:
+
+stdenv.mkDerivation rec {
+  pname = "phd2";
+  version = "2.6.9dev1";
+
+  src = fetchFromGitHub {
+    owner = "OpenPHDGuiding";
+    repo = "phd2";
+    rev = "v${version}";
+    sha256 = "1ih7m9lilh12xbhmwm9kkicaqy72mi3firl6df7m5x38n2zj3zm4";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig wrapGAppsHook ];
```

Comment:
```suggestion
  nativeBuildInputs = [ cmake pkg-config wrapGAppsHook ];
```

Reply:
Changed in accordance with recommendation.